docs/UserGuide.md

```diff
@@ -175,14 +175,14 @@ __Optional Parameter 1: Gender__
 Format: `edit INDEX [g/GENDER]`
 
 - Edits the gender of a person in the contact list. The genders accepted by the contact list are: `M`/`m` for male,
-`F`/`f` for female, and `NA` for empty gender (if user wants to hide gender).
+`F`/`f` for female.
 - `INDEX` must be **a positive integer** (i.e 1,2,3…)
 - `INDEX` must be within the range of the contact list index (i.e. from 1 to size of contact list).
 
 Examples:
 * `edit 1 g/M`
 * `edit 2 g/f`
-* `edit 3 g/NA`
+* `edit 3 g/Female`
```

Comment:
This is not consistent with your guide above as you indicated that the genders accepted are `F`/`f` for female.